.github/workflows/releasecandidate.yml

```diff
@@ -67,3 +67,8 @@ jobs:
           slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
           channel: '#team-devops'
           name: 'Workflow Status'
+
+  run-sdk-integration-networking-tests:
+    needs: [staticbuild, appimage]
+    if: ${{ success() }}
+    uses: ./trigger-networking-tests.yml
```

Comment:
can also re-integrate this workflow back into the rc workflow, but this current set-up makes it reusable by other workflows if needed in the future